docs/source/benchmarking/materials_science_and_engineering/oiemodels.rst

```diff
@@ -1,12 +1,12 @@
 Open Innovation Environment (OIE) domain ontologies, Models module (OIEModels)
```

Comment:
convert `Open Innovation Environment (OIE) domain ontologies, Models module (OIEModels)` to `Open Innovation Environment Models (OIEModels)`